code/modules/mob/living/simple_animal/friendly/sloth.dm

```diff
@@ -37,6 +39,15 @@
 /mob/living/simple_animal/sloth/Initialize(mapload)
 	. = ..()
 	AddElement(/datum/element/pet_bonus, "slowly smiles!")
+	// If someone adds non-cargo sloths to maps we'll have a problem but we're fine for now
+	if(!GLOB.cargo_sloth)
+		GLOB.cargo_sloth = src
+
+/mob/living/simple_animal/sloth/Destroy()
+	if(GLOB.cargo_sloth == src)
```

Comment:
If they give cargo’s sloth then get another, that will become cargo’s sloth?

Reply:
i assume you mean "gib" and yes, for all intents and purposes